migrations/sql/07_pg_stat_monitor_columns.up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE metrics
+  ADD COLUMN `client_ip` LowCardinality(String),
```

Comment:
I think this is duplicate column 
https://github.com/percona/qan-api2/blob/master/migrations/sql/01_init.up.sql#L8
> `client_host` LowCardinality(String) COMMENT 'client IP or hostname',

Reply:
good point, thank you.